biosimulations/apps/dispatch-service/src/app/services/hpc/hpc.service.ts

```diff
@@ -33,18 +33,18 @@ export class HpcService {
           'Simdirectory created on HPC: ' + JSON.stringify(value)
         );
 
-        this.sshService
-          .putFile(omexPath, `${simDirBase}/in/${omexName}`)
-          .then((val) => {
-            this.logger.log(
```

Comment:
Lets remove instead of commenting out. We have the git history if it is needed

Reply:
Sure, will do it.